### ISSUE TITLE
12.3.5 Open Information and Open Data

### ISSUE THREAD

**Issue by AlexanderRitchie** [bcgov/digital-policy#5]
When I read the requirements under this section, I feel that there is a conflation of 'Open' with 'sharing' of data and information.

Open generally means unrestricted and unencumbered access public access to data and information with appropriate licensing (such as the [OGL-BC](https://www2.gov.bc.ca/gov/content/data/open-data/open-government-licence-bc)). Sharing in the context as I read it refers to the ability for ministries and partners to access data to and from each other in a way to increase the value and use of the data.

A few suggestions:
1) To reduce the conflation of topics, I think that the concept of sharing could fit into 12.3.4 by increasing scope to Enterprise solutions including re-use and sharing of data (it is lightly mentioned in data registers)
2) While recognition of open is important - I suggest it be contextualized as open with a purpose and defined value. There is a cost associated with 'efforts to expand public access' and without additional resources, it is difficult and may not derive value. There can be cynicism and resultant negative effect of statements that 'ministries must' without providing the necessary resources to meet the need. I recommend a structure of proportionality or a spectrum of open and access as there can be value of open to portion of data and information without the challenges that setting the goal post of full open can lead to, which again can have negative consequences and limiting any action.

**Comment by glam-bert** [comment]
Agree, it would be helpful to be more explicit about supporting the proportionate governance of data, with open data at one end of the continuum, and other approaches (such as data commons) that enable data to be safely shared where more controls are required.

**Comment by HeatherRemacle** [comment]
I am wondering about open code (considered information?)... and offer link to the Government of Canada's guide on this: https://www.canada.ca/en/government/system/digital-government/open-source-software/guide-for-publishing-open-source-code.html

Excerpt:

"As such it is recommended that where they have the right to do so, departments publish all source code as open source software, whether the software solution was (i) acquired as OSS; (ii) developed in-house by GC employees or (iii) acquired through the terms of procurement contracts where appropriate license terms were negotiated.

When releasing the code at large is not appropriate, or possible, it is recommended to start with sharing the source code internally to all departments, to the extent that the terms of the applicable license permit such sharing. Care should be taken to ensure that the license to Canada does not limit the distribution of such code to specified user departments. Doing so will help make the source code ready to be released publicly as open source where Canada has received the right to do so."
